### PR TITLE
Added header to all pages

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Head, header, headest :+1:

Fixes JanKoehnlein/demo#3

### CODE REVIEW

File: hello-next/pages/index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+import Header from "../components/header"
+
 const Index = () => (
   <div>
+    <Header />
```
Comment (JanKoehnlein) [inline hello-next/pages/index.js]: Or should be do a footer instead?

Comment (JanKoehnlein) [inline hello-next/pages/index.js]: Should it really be inside the `div`

Comment (JanKoehnlein) [inline hello-next/pages/index.js:5]: Should it really be inside the `div`

Comment (JanKoehnlein) [reply hello-next/pages/index.js]: Yes, I think so

Comment (JinKoehnlein) [reply hello-next/pages/index.js:5]: Yes, I think so

**Review by JinKoehnlein — APPROVED** [review]
Looks good to me :+1:

**Pull request closed without merging**